tests/strategies/pilot_users_unit_test.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from ioet_feature_flag.strategies import PilotUsers
+from ioet_feature_flag.exceptions import MissingToggleAttributes
+from ioet_feature_flag.toggle_context import ToggleContext
+
+
+class TestCutoverStrategy:
+    @pytest.mark.parametrize(
+        "is_enabled, current_user, allowed_users, expected_result, expected_exception",
+        [
+            (True, 'allowed_user', '', False, MissingToggleAttributes),
+            (True, 'allowed_user', 'allowed_user', True, None),
+            (True, 'allowed_user', 'allowed_user,another_user', True, None),
+            (False, 'allowed_user', 'allowed_user', False, None),
+            (False, 'allowed_user', 'allowed_user,another_user', False, None),
+        ]
+    )
+    def test__returns_toggles_specified_in_attributes(
```

Comment:
What should be the behaviour if a user does not appear in the allowed users? For example, on the following scenarios:
- If the current user is `'not_allowed_user'` and the allowed user is `'allowed_user'` and the attribute enabled is set to `True` then should the expected return `False` or an Exception?
- With the same preconditions stated above, if the attribute enabled is set to `False`, then this `'not_allowed_user'` should the expected result return `True` or an exception?

Reply:
It just returns `False` for both of those cases, I realized that I didn't add them to the test cases :see_no_evil: let me add them